public/posts/registry2.md

```diff
@@ -0,0 +1,70 @@
+Since the beginning the goal for deno.land/x is to give all modules in the Deno
+ecosystem a reliable, easy to use, and free host. With the increased adoption of
+Deno in the recent month there have been more and more requests to add features
+like download counts, immutable versions, and a self service module upload.
+
+We also have some long standing issues like rate limits on the GitHub API that
+we use, and `https://deno.land/std` being an alias to
+`https://deno.land/std@master` rather than the latest version. These were issues
+we could not adress without a redesign of deno.land/x.
+
+## Issues to solve
+
+The main issues we wanted to address in this first stage were:
+
+- making module upload self service
+- making all versions immutable
+- removing non versioned imports
+- and removing the GitHub API rate limits on the website
+
+To do this we setteled on a design where all source code is stored our servers,
+and we retrieve a new version from GitHub whenever it is published. We get
+notified of this by a GitHub Webhook you add to your repository.
+
+## What has changed?
+
+- Source code is now not stored on raw.githubusercontent.com, rather on our own
```

Comment:
`Source code is no longer stored on raw.githubusercontent.com but rather on our own servers`